BigFileMergeSort/src/ua/goit/alg/TestBigFileMergeSort.java

```diff
@@ -0,0 +1,61 @@
+package ua.goit.alg;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.DataInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class TestBigFileMergeSort {
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+    static String property = "java.io.tmpdir";
+    static String tempDir = System.getProperty(property);
+
+    @Test
+    public void test() throws IOException {
+
+	File file = folder.newFile("file.txt");
+	FileWriter fw1 = new FileWriter(file);
+	String cq1 = "9 9 9 1 3 5 7 2 3 5 8 1 8 5 3 1 8 4 5 7";
+	fw1.append(cq1);
+	fw1.flush();
+	fw1.close();
+	File fileExpected = folder.newFile("expectedfile.txt");
```

Comment:
test input creation can be refactored to a function (s) for easier readability. I can see a lot of duplicated code around